templates/views/views-view--contacts-a-z.html.twig

```diff
@@ -55,7 +55,7 @@
     {% endif %}
 
     {% if links %}
-      <div class="view-links" aria-label="{{ 'search options'|t }}">
+      <div class="view-links" id="contact_links" aria-label="{{ 'search options'|t }}">
```

Comment:
@David-Corbett @neilblair do you guys prefer to use classes vs id's in your markup?